R/do_adjustRtime-functions.R

```diff
@@ -874,7 +874,7 @@ NULL
                       resid_ratio = 3,
                       zero_weight = 10,
                       bs = "tp"){
-    rt_map <- rt_map[order(rt_map$obs), ]
+    rt_map <- rt_map[order(rt_map$obs), c("ref", "obs")]
```

Comment:
was this not added in a previous push ? @jorainer

Reply:
same with the vignette update ?

Reply:
honestly - I also got a bit confused what we have now in the devel branch and what not...

Reply:
hm, maybe it's then in one of the other open pull requests?

Reply:
yup that's exactly that ! I'll fix it if your PR gets merge first